plugins/code-climate/src/components/CodeClimateTable/CodeClimateTable.tsx

```diff
@@ -0,0 +1,212 @@
+    display: 'flex',
+    justifyContent: 'space-around',
+  },
+  spaceBetween: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
+  maintainabilityLetterColor: {
+    ...letterStyle,
+    backgroundColor: props =>
+      letterColor((props as any).maintainabilityLetter as string),
+  },
+  testCoverageLetterColor: {
+    ...letterStyle,
+    backgroundColor: props =>
+      letterColor((props as any).testCoverageLetter as string),
+  },
+  fontSize: {
+    ...fontSize,
+  },
+  letterDetails: {
+    ...fontSize,
+    paddingLeft: '10px',
+  },
+  paddingTop20: {
+    paddingTop: '20px',
+  },
+  paddingTop30: {
+    paddingTop: '30px',
+  },
+  paddingSides20: {
+    padding: '0px 20px',
+  },
+});
+
+export const CodeClimateTable = ({
+  codeClimateData,
+}: CodeClimateTableProps) => {
+  const {
+    repoID,
+    maintainability: {
+      letter: maintainabilityLetter,
+      value: maintainabilityValue,
+    },
+    testCoverage: { letter: testCoverageLetter, value: testCoverageValue },
+    numberOfCodeSmells,
+    numberOfDuplication,
+    numberOfOtherIssues,
+  } = codeClimateData as CodeClimateData;
+
+  const classes = useStyles({ maintainabilityLetter, testCoverageLetter });
+
+  if (!codeClimateData) {
+    return null;
+  }
+
+  return (
+    <InfoCard title="Code Climate Summary">
+      <div className={classes.spaceAround}>
+        <a href={`https://codeclimate.com/repos/${repoID}`}>
+          <Typography variant="h6" component="p">
+            Maintainability
+          </Typography>
+          <div className={classes.spaceBetween}>
+            <Typography
+              className={classes.maintainabilityLetterColor}
+              variant="body2"
+              component="p"
+            >
+              {maintainabilityLetter}
+            </Typography>
+            <Typography
+              className={classes.letterDetails}
+              variant="body2"
+              component="p"
+            >
+              {maintainabilityValue}
+            </Typography>
+          </div>
+        </a>
+        <a href={`https://codeclimate.com/repos/${repoID}`}>
+          <Typography variant="h6" component="p">
+            Test Coverage
+          </Typography>
+          <div className={classes.spaceBetween}>
+            <Typography
+              className={classes.testCoverageLetterColor}
+              variant="body2"
+              component="p"
+            >
+              {testCoverageLetter}
+            </Typography>
+            <Typography
+              className={classes.letterDetails}
+              variant="body2"
+              component="p"
+            >
+              {testCoverageValue}%
+            </Typography>
+          </div>
+        </a>
+      </div>
+      <div className={clsx(classes.spaceAround, classes.paddingTop30)}>
+        <div>
+          <Typography variant="h6" component="p">
+            Code Smells:
+          </Typography>
+          <a
```

Comment:
Please always use `Link` from `@backstage/core-components` instead of `a` - it comes with a lot of benefits such as consistent open-in-new-tab behavior, click tracking, security features, etc